docs/releases/index.rst

```diff
@@ -14,9 +14,7 @@ If you are upgrading Pootle, you might want to see some tips to ensure your
 .. toctree::
    :maxdepth: 1
 
-   2.8.0b3 <2.8.0b3>
-   2.8.0b2 <2.8.0b2>
-   2.8.0b1 <2.8.0b1>
+   2.8.0
```

Comment:
Yes it is, they work just like normal links. Its just using the header form the 2.8.0.rst file